src/fparser/two/Fortran2008.py

```diff
@@ -80,43 +80,52 @@
 
 from fparser.common.splitline import string_replace_map, splitparen
 from fparser.two import pattern_tools as pattern
-
 from fparser.two.utils import (
     BracketBase,
     CALLBase,
     KeywordValueBase,
     NoMatchError,
+    ScopingRegionMixin,
     SeparatorBase,
     StmtBase,
     STRINGBase,
     Type_Declaration_StmtBase,
     WORDClsBase,
 )
+
+# The _List classes imported here are auto-generated which confuses pylint
+# (because it doesn't actually import modules and therefore the classes are
+# not generated).
+# pylint: disable=no-name-in-module
```

Comment:
This was added because I asked for it in a previous comment, but if we bring this to master now with the new pylintrc, do we still need it?
If fact can you grep for the new pylintrc disabled messages and see if we can remove some more inlined disable statements.

Reply:
It's gone now. (The pylint documentation is really bad!)